packages/browser-wallet-api-helpers/README.md

```diff
@@ -147,3 +147,12 @@ provider.on('accountDisconnected', (accountAddress) => {
 // Connect to an account in the wallet again triggered elsewhere in the dApp.
 provider.connect().then((accountAddress) => (selectedAccountAddress = accountAddress));
 ```
+
+## Accessing node through JSON-RPC
+
+The wallet API exposes access to a JSON-RPC client. This allows a dApp to communicate with the same node as the wallet is connected to, and enables dApps to access the JSON-RPC interface without being connected to a separate server itself. The client is accessed as shown in the example below.
+
+```typescript
+const provider = await detectConcordiumProvider();
+const accountInfo = await provider.getJsonRpcClient().getAccountInfo(accountAddress);
```

Comment:
I think the example should show how to get the client, and then it should also state that the client will be outdated if the network changes.